packages/sakuli-cli/src/commands/enable-enterprise/tasks/license-tasks.function.spec.ts

```diff
@@ -1,20 +1,21 @@
 import { licenseGlobalTask } from "./license-tasks.function";
-import execa = require("execa");
 import { promises as fs } from 'fs';
-import { sep, join } from "path";
+import { join, sep } from "path";
 import { homedir } from "os";
+import execa = require("execa");
 import rimraf = require("rimraf");
+
```

Comment:
It's so annoying to see "changes" automatically made by IntelliJ's code cleaning (even if they are useful)... 
Hope we can give #125 a change in the future to have common formatted codebase :)

(Sorry the small rant does not relate to you, I just want to advertise issue #125 ;))